### PR TITLE
add maps

### PR DESCRIPTION
**Pull request by hyuunnn**

### CODE REVIEW

**Comment by hyuunnn** [comment]
Microsoft-Windows-Windows-Firewall-With-Advanced-Security_2003.map
image
01000000 -> 01-00-00-00 (Enable)
00000000 -> 00-00-00-00 (Disable)

File: evtx/Maps/Application_System Restore_8194.map [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Author: Hyun Yi @hyuunnn
+Description: restore point created successfully
```
Comment (rathbuna) [inline evtx/Maps/Application_System Restore_8194.map:2]: Restore, not restore. Can you capitalize that first word?

**Review by rathbuna — CHANGES_REQUESTED** [review]

**Comment by rathbuna** [comment]
Looking great. Nice work.

File: evtx/Maps/Application_System Restore_8194.map (hunk 1 above)
Comment (hyuunnn) [reply evtx/Maps/Application_System Restore_8194.map]: @rathbuna Fixed :)

**Pull request merged**